### ISSUE TITLE
Need to unwrap optionals in queries of optional KeyPaths

### ISSUE THREAD

**Issue by schrockblock** [MihaelIsaev/FluentQuery#9]
I have a class with an optional column, something like this:
```
final class SomeRecord: Codable, PostgreSQLModel, Content {
  // ...
  var string: String?
  // ...
}
```
Unfortunately because the column is optional, doing this:
```
let query = FQL().select("*").from(SomeRecord.self).where(\SomeRecord.string != "value")
```
results in this SQL:
```
SELECT * FROM _some_records_ WHERE "_some_records_"."string" != 'Optional("value")'
```
I have a suspicion of where the issue is occurring (the `formatValue` function in `FQPredicate`, in the `String` branch of the `switch` statement, or perhaps just the `singleQuotted(v: Any)` function which doesn't understand optionals) ...might open a PR to fix it.

**Comment by MihaelIsaev** [comment]
Thanks for catching this issue. It'd be great if you could send a pull request to fix this 🙂 

Since `FluentQuery` isn't protected from SQL-injections may I suggest you to switch to `SwifQL`? 🙂 
Your query in `SwifQL` will look like
```swift
SwifQL.select(SomeRecord.table.*).from(SomeRecord.table).where(\SomeRecord.string == "value")
```

**Comment by schrockblock** [comment]
Thanks @MihaelIsaev – I could be convinced to move over to SwifQL, but when I last looked it didn't support `WINDOW` or `PARTITION`. Does it support those now?

And interestingly, I just forked so I could submit the changes, but your master branch has exactly what I would do: use `String(describing:)` in `StringExtensions`. You can see that here:

https://github.com/MihaelIsaev/FluentQuery/commit/442b30f24dbbf44f1ac0f3fec78274810de4dc70

Does the current release include that commit? Was it changed back for some reason?

**Comment by MihaelIsaev** [comment]
> Was it changed back for some reason?
Current `master` even supports MySQL, but it was never released.

> when I last looked it didn't support WINDOW or PARTITION. Does it support those now?
No, but you could easily add them through extensions, and please feel free to send pull request.

Please take a look at `SwifQLable+Select`, `SwifQLable+Union` and other files in `SwifQL` and you'll see how easy it is to extend it! 🙂

**Comment by schrockblock** [comment]
@MihaelIsaev I may at a future time, but for now just opened a PR to fix this issue. Thanks for responding so quickly!

**Comment by MihaelIsaev** [comment]
Thanks for the fix! Now it's available in `v0.4.32` 🎉
